### PR TITLE
RSWEB-7525: remove 960 global elements

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-7525
Removing 960 from global elements.
The rug was a little trickier so we need to maybe add one or two things to rug.

### CODE REVIEW

File: styleguide/_themes/derek/scss/globalElements/rug.scss [hunk 1]
```diff
@@ -39,12 +42,13 @@
...
+  float: left;
+  padding: 7px 15px;
 }
 
 .rug-subhead {
   color: $white;
-  font-size: 37px;
-  line-height: 24px;
+  font-size: 43px;
+  line-height: 30px;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/globalElements/rug.scss]: Since we're changing values anyway, can we use `rem` here instead of pixels?

Comment (amberRrucker) [inline styleguide/_themes/derek/scss/globalElements/rug.scss]: I did the thing.

**Comment by xinbin** [comment]
Please rebase. <3  so I can rebuild this in a test branch

**Comment by WidgetsBurritos** [comment]
This PR too?

**Pull request closed without merging**